db/migrate/20240609101057_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
```

Comment:
@udk-wtag bhai, according to the migration file, are we allowing `null` values for `name` and `email` at the db level?

Reply:
@sal-wtag bhai, I have only added validation at model level. I am working on to add validation also at db lebel.